pyulog/__init__.py

```diff
@@ -1,5 +1,7 @@
 """ Wrapper to include the main library modules """
 from .core import ULog
+from . import ulog2kml
+from . import ulog2csv
```

Comment:
I prefer to have only the core components imported by default, as others might pull in additional optional dependencies (as is the case for ulog2kml).

Reply:
Noted. Reverted the imports back to original.